cmd/uhc/cluster/list/list.go

```diff
@@ -220,11 +298,53 @@ func run(cmd *cobra.Command, argv []string) {
 
 		// Display the fetched page:
 		response.Items().Each(func(cluster *v1.Cluster) bool {
-			// Get cluster API path
+
+			// If the custom columns flag was raised,
+			// populate cluster array with custom columns:
+			if args.columns != "" {
+
+				// String to output marshal -
+				// Map used to parse Cluster data -
+				// Writer to body variable:
+				var body string
+				var jsonBody map[string]interface{}
+				boddyBuffer := bytes.NewBufferString(body)
+
+				// Write Cluster data to body variable:
+				err := v1.MarshalCluster(cluster, boddyBuffer)
+				if err != nil {
+					fmt.Fprintf(os.Stderr, "Failed to Marshal Cluster into byte buffer: %s\n", err)
```

Comment:
Remeber to write messages with proper use of upper and lower case: "Marshal" and "Cluster" should be lower case, in my opinion.